### PR TITLE
[FIX] Password reset message

### LINKED ISSUE #7339: Lost password's "password sent" action UX flow updates
Some minor things I've noticed about the change password handling:

### Don't show the confirmation page unless there's something to confirm

If I access `/Security/lostpassword/passwordsent` I get shown:

image

You can also add `/foo` etc to the end to get it to show any username, which is also not necessary or desirable.

image

### Don't include a period inside the username/email address

I assume that the purpose of putting the username/email address in quotes is to highlight it as a username/email address. If this is the case, the period should not be inside the quotes.

I suppose grammatically speaking that `Password reset link sent to 'foo@example.com.'` is correct, but `A reset link has been sent to 'foo@example.com.', provided...` is not.

My suggestion here would be to just remove the period entirely.

### PR DESCRIPTION
**Pull request by Firesphere**
Fixes #7339

### CODE REVIEW

**Comment by Firesphere** [comment]
I don't know how @tractorcow his commits got in here, I branched directly off origin/4

**Comment by tractorcow** [comment]
Use 4.0 not 4 please. :)

**Comment by Firesphere** [comment]
👍 Yep, will do :)

File: composer.json [hunk 1]
```diff
@@ -61,6 +61,10 @@
         "psr/container-implementation": "1.0.0"
     },
     "extra": {
+        "branch-alias": {
```
Comment (tractorcow) [inline composer.json]: This commit needs deleting from this PR.

Comment (Firesphere) [reply composer.json]: I have no idea how it got in there. Dropped from PR

**Pull request merged**